client/src/ui/organisms/footer/index.tsx

```diff
@@ -78,6 +78,7 @@ export function Footer() {
                 href="https://twitter.com/mozdevnet"
                 target="_blank"
                 rel="noopener noreferrer"
+                aria-label="Follow MDN on Twitter"
```

Comment:
Not to open a can of worms: I think on Kuma these only says "Twitter" which nicely ditches the l10n issue here.